otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -145,6 +146,10 @@ def _init_task(self, dataset: Optional[DatasetEntity] = None, export: bool = Fal
         # Update recipe with caching modules
         self._update_caching_modules(self._recipe_cfg.data)
 
+        # Loss dynamics tracking
```

Comment:
As we decide to collect inner patch of config to configurer, this should be moved to DetectionConfigurer. 
I'd like to suggest to add enable_noisy_label_detection flag in patch_from_hyperparams function. patch_from_hyperparams is for merging config and hyperparameters. After that DetectionConfigurer should patch config when cfg.enable_noisy_label_detection is True